.github/workflows/CI.yml

```diff
@@ -93,10 +93,14 @@ jobs:
         shell: bash -l {0}
         run: pylint --extension-pkg-whitelist=netCDF4 --ignored-modules=umpost -E umpost
 
-      - name: Entrypoint test
+      - name: Entrypoint test of driver script
         shell: bash -l {0}
         run: esm1p5_convert_nc --help
 
+      - name: Entrypoint test of um2netcdf script
+        shell: bash -l {0}
```

Comment:
Should we invoke the shell with an `-e` flag so it errors on failure?

Reply:
Is using `-e` the practice in other ACCESS repos?

Reply:
I often just use the [default (bash) shell](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) (no `shell: *` section at all unless required) - which is `bash --noprofile --norc -eo pipefail {0}`

Reply:
Ok, to clarify, do you simply specify it as this?
```YAML
- name: Entrypoint test of um2netcdf script
  shell: bash
  run: um2nc --help
```

Reply:
Even simpler, you can do:
```
- name: Entrypoint test of um2netcdf script
  run: um2nc --help
```
But now I wonder if they're specified for a reason...

Reply:
It may have been from copying the CI config from another repo. @aidanheerdegen @blimlim do either of you recall if the test config was copied in, or was there was a reason for adding `bash -l {0}`?

Reply:
The login requirement for the shell was a recommendation from the miniforge action docs, otherwise the conda stuff wasn't set up correctly. I did wonder about making it default, and wasn't sure about the `-e`, so feel free to add that at the very least, or set up a default to do so. You might need to make a specific suggestion if you think we should make that change @CodeGat, or park it and make an issue to change it in the future.

Reply:
https://github.com/conda-incubator/setup-miniconda#use-a-default-shell

Reply:
@aidanheerdegen @blimlim @CodeGat  To help with getting the recent entry point changes in for the release, are folks OK with moving the alternate `bash` suggestion to another issue & using these working settings for now?

Reply:
Yep park it :fire: